requirements/requirements.txt

```diff
@@ -15,15 +15,9 @@ ipython
 tensorboard>=1.14.0
 tensorboardX
 crc32c>=1.7
+opencv-python-headless
```

Comment:
why do we need `opencv` still?

Reply:
tensorboard - https://github.com/catalyst-team/catalyst/blob/9f97ad91c82322a18ac6a5764aaded7fc54720a4/catalyst/contrib/utils/tools/tensorboard.py#L160

Reply:
Looks like `SummaryReader` does not use this feature,
- it is used only for scalar logs reading
- by default it does not support image handling
- we don't have a need to read tensorboard image-based logs (usually all you need are scalar one)

What do you think about removing this feature?

Reply:
and what about confusion matrix?

Reply:
done